### ISSUE TITLE
pctl version isn't correct

### ISSUE THREAD

**Issue by aclevername** [weaveworks/pctl#240]
From the latest `v0.2.0` release:
```
$  pctl --help
NAME:
   pctl - A cli tool for interacting with profiles

USAGE:
   pctl [global options] command [command options] [arguments...]

VERSION:
   0.1.0-dev+6b75a51.2021-07-28T13:14:29Z
   ....
   
$ pctl --version
pctl version 0.1.0-dev+6b75a51.2021-07-28T13:14:29Z
```

**Comment by Himangini** [comment]
you need to update the `release.go` file I think with the new tagged version. this should be fixed with #233

**Comment by aclevername** [comment]
ahhh. @Himangini can you open a quick PR to add that to the release process docs? https://github.com/weaveworks/pctl#release-process . Just to tide us over until thats done :smile:
